packages/components/psammead-grid/src/index.jsx

```diff
@@ -0,0 +1,235 @@
+import React from 'react';
+import { node } from 'prop-types';
+import styled, { css } from 'styled-components';
+import {
+  GEL_MARGIN_BELOW_400PX,
+  GEL_GUTTER_BELOW_600PX,
+  GEL_MARGIN_ABOVE_400PX,
+  GEL_GUTTER_ABOVE_600PX,
+} from '@bbc/gel-foundations/spacings';
+import {
+  // GEL_GROUP_0_SCREEN_WIDTH_MIN,
+  GEL_GROUP_0_SCREEN_WIDTH_MAX,
+  GEL_GROUP_1_SCREEN_WIDTH_MIN,
+  GEL_GROUP_1_SCREEN_WIDTH_MAX,
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_2_SCREEN_WIDTH_MAX,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MAX,
+  GEL_GROUP_5_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+// import { grid } from '@bbc/psammead-styles/detection';
+
+const groups = {
```

Comment:
I was thinking.. could we add in this object, also the group respective `gutter` and `margin` instead of defining separately `gelGridGutters` and `gelMargins`? And then print the values inside the `gridMediaQueries`?

Otherwise, for cases like this one: 
`<Grid wrapper enableGelGutters enableGelMargins columns={{ group4: 8 }}>`
we will be adding the `grid-column-gap` and `padding` for other breakpoints, rather than just for the one defined in by the `group4`.

Reply:
Yes, that's a good point. We should still be able to toggle them on/off, just the implementation behind the scenes should be simpler, as you suggested. I'll see if I can make the change prior to the review session!